.github/CODEOWNERS

```diff
@@ -15,28 +15,8 @@
 
 ### ADAPTERS
 
-# Adapter interface ("base" + "sql" adapter defaults, cache)
-/core/dbt/adapters              @dbt-labs/core-adapters
-
-# Global project (default macros + materializations), starter project
-/core/dbt/include               @dbt-labs/core-adapters
```

Comment:
this still exists but is just the starter_project for dbt init, shouldn't be an adapters concern any longer

https://github.com/dbt-labs/dbt-core/tree/main/core/dbt/include